.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
-* text=auto
+* text eol=lf
 
 # Declare files that will always have CRLF line endings on checkout.
-*.prp text eol=crlf
\ No newline at end of file
+*.prp text eol=crlf
```

Comment:
Only for the .prp files. And that is to ensure that if you build a release on any platform, the config.prp files must be readable for any user. This can only be ensured by enforcing `crlf` for the config.prp files.